pkg/profiler/cpu/maps.go

```diff
@@ -1604,11 +1663,11 @@ func (m *bpfMaps) setUnwindTableForMapping(buf *profiler.EfficientBuffer, pid in
 			restChunks = restChunks[threshold:]
 
 			if currentChunk[0].IsEndOfFDEMarker() {
-				level.Error(m.logger).Log("msg", "First row of a chunk should not be a marker")
```

Comment:
wondering why the linter never complain about these ones

Reply:
🤷 We need to revisit our configuration.